.github/workflows/functional-tests.yml

```diff
@@ -109,13 +96,13 @@ jobs:
 
       - name: Post workflow result
         id: slack
-        if: ${{steps.runFunctionalTests1.outcome == 'failure'}}
+        if: ${{steps.runFunctionalTests.outcome == 'failure'}}
         uses: slackapi/slack-github-action@v1.25.0
         with:
           payload: |
             {
-              "functional_test_retried": "${{steps.runFunctionalTests1.outcome == 'failure'}}",
-              "workflow_failed": "${{steps.runFunctionalTests1.outcome == 'failure' && steps.runFunctionalTests2.outcome == 'failure'}}"
+              "functional_test_retried": "${{steps.runFunctionalTests.outcome == 'failure'}}",
```

Comment:
We can remove this line as we are not retrying anymore.

Reply:
Removed this line.

Reply:
Also it would be good to change the comparison to failure and send a positive variable

Reply:
Maybe we can rename from `workflow_failed ` to `workflow_succeeded`